document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/ProsecutionDocumentInfoService.java

```diff
@@ -49,140 +47,31 @@
 public class ProsecutionDocumentInfoService implements DocumentInfoService {
     public static final String MODULE_NAME_SPACE = "document-generator-prosecution";
 
-    private static final String LOGGING_RESOURCE_KEY = "resource";
-
     private static final Logger LOG = LoggerFactory.getLogger(MODULE_NAME_SPACE);
 
-    private static final String MATCH_START_OF_SJP_URI = "/prosecution/sjp/";
-
-    private static final String MATCH_START_OF_ULTIMATUM_URI = "/prosecution/ultimatum/";
-
-    private static final int ULTIMATUM_INDEX_TO_TRIM_FROM =
-                    MATCH_START_OF_ULTIMATUM_URI.length() - 1;
-
-    private static final int SJP_INDEX_TO_TRIM_FROM = MATCH_START_OF_SJP_URI.length() - 1;
-
-    private final UltimatumDocumentInfoBuilderProvider docInfoBuilderProvider;
-
-    private final ProsecutionClient prosecutionClient;
-
-    /**
-     * Constructor, which will be autowired.
-     * 
-     * @param docInfoBuilderProvider
-     * @param prosecutionClient
-     */
-    public ProsecutionDocumentInfoService(
-                    UltimatumDocumentInfoBuilderProvider docInfoBuilderProvider,
-                    ProsecutionClient prosecutionClient) {
-        this.docInfoBuilderProvider = docInfoBuilderProvider;
-        this.prosecutionClient = prosecutionClient;
-    }
+    @Autowired private ProsecutionHandler handler;
```

Comment:
Please fix the formatting to be consistent

Reply:
Okay, will do